misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/variable/types/BString.java

```diff
@@ -29,14 +29,17 @@
  */
 public class BString extends BSimpleVariable {
 
+    private static final String SYMBOL_DOUBLE_QUOTE = "\"";
+
     public BString(SuspendedContext context, String name, Value value) {
         super(context, name, BVariableType.STRING, value);
     }
 
     @Override
     public String computeValue() {
         try {
-            return getStringFrom(jvmValue);
+            // Add double quote to the beginning and end of the computed string value.
+            return SYMBOL_DOUBLE_QUOTE.concat(getStringFrom(jvmValue)).concat(SYMBOL_DOUBLE_QUOTE);
```

Comment:
Shall we use `+` operator instead of `concat` for better readability and performance?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/31606/commits/6e1c73b663b987efcbfb95e4af91ded03ced36a8